kylecornelissen/vacation-rentals
# Area listings

What's this PR do?
- Adding a ListingsContainer for each Area
- Button added to each Area in `/areas` that routes to ListingsContainer
- Change listing attribute to listings

How should this be manually tested?
- fill out both inputs at `/` and enter site. Click any Listings button on an Area
- Should navigate to `/areas/:id/listings` and show only that area's listings

What are the relevant tickets?  
#11 

Future Implementation Plans:
- Add info to each listing card
- Make sure navigation bar persists through each page (maybe not the first page though)
Questions:
